Editor/Menus/CinemachineMenu.cs

```diff
@@ -97,14 +100,23 @@ static void CreateClearShotVirtualCamera(MenuCommand command)
         static void CreateDollyCameraWithPath(MenuCommand command)
         {
             CinemachineEditorAnalytics.SendCreateEvent("Dolly Camera with Track");
-            var path = CreateCinemachineObject<CinemachineSmoothPath>(
-                "Dolly Track", command.context as GameObject, false);
             var vcam = CreateCinemachineObject<CinemachineVirtualCamera>(
                 "Virtual Camera", command.context as GameObject, true);
             vcam.m_Lens = MatchSceneViewCamera(vcam.transform);
-
             AddCinemachineComponent<CinemachineComposer>(vcam);
+#if CINEMACHINE_UNITY_SPLINES
+            var splineContainer = ObjectFactory.CreateGameObject("Dolly Track", typeof(SplineContainer)).GetComponent<SplineContainer>();
+            splineContainer.Spline.EditType = SplineType.CatmullRom;
+            splineContainer.Spline.Add(new BezierKnot(Vector3.zero));
+            splineContainer.Spline.Add(new BezierKnot(Vector3.right));
+            AddCinemachineComponent<CinemachineSplineDolly>(vcam).m_Spline = splineContainer;
+#else
+            var path = CreateCinemachineObject<CinemachineSmoothPath>(
+                "Dolly Track", command.context as GameObject, false);
+#pragma warning disable 618 // disable obsolete warning
             AddCinemachineComponent<CinemachineTrackedDolly>(vcam).m_Path = path;
+#pragma warning restore 618
+#endif
         }
 
         [MenuItem(m_CinemachineGameObjectRootMenu + "Dolly Track with Cart", false, m_GameObjectMenuPriority)]
```

Comment:
This menu entry still gives the old-fashioned path.  What should we do about this?

Reply:
We could remove it, when splines are installed. It is discoverable in our examples. In CM 3, we could remove it completely.